{
  "WorkItem": {
    "AffectedComponent": {
      "Name": "",
      "DisplayName": ""
    },
    "ClosedComment": "",
    "ClosedDate": null,
    "CommentCount": 0,
    "Custom": null,
    "Description": "The following python 2.5 script:\n \nimport smtplib\ns=smtplib.SMTP('localhost',25)\ntolist=['one@one.org']\nmsg='''\nFrom: Me@my.org\nSubject: testin'...\n \nThis is a test '''\ns.sendmail(\"me@my.org\",tolist,msg)\ns.quit()\n \n \nGenerates the following smtp4dev log:\n \n220 localhost smtp4dev ready\nehlo [10.0.5.33]\n250-Nice to meet you.\n250-8BITMIME\n250-AUTH=CRAM-MD5 PLAIN LOGIN ANONYMOUS\n250-AUTH CRAM-MD5 PLAIN LOGIN ANONYMOUS\n250 SIZE\nmail FROM:<me@my.org> size=53\n501 Parameter size is not recognised\nrset\n \n \nAll options in smtp4dev are defaults.",
    "LastUpdatedDate": "2013-06-14T00:31:49.137-07:00",
    "PlannedForRelease": "smtp4dev 2.0.8",
    "ReleaseVisibleToPublic": true,
    "Priority": {
      "Name": "High",
      "Severity": 150,
      "Id": 3
    },
    "ProjectName": "smtp4dev",
    "ReportedDate": "2010-09-07T09:32:39.897-07:00",
    "Status": {
      "Name": "Resolved",
      "Id": 7
    },
    "ReasonClosed": {
      "Name": "Unassigned"
    },
    "Summary": "501 Parameter size is not recognised",
    "Type": {
      "Name": "Issue",
      "Id": 3
    },
    "VoteCount": 1,
    "Id": 5557
  },
  "FileAttachments": [],
  "Comments": [
    {
      "Message": "Doh! looks like I've accidentally made the parameter names case sensitive and it will currently only accept them in upper case e.g. SIZE.\r\nI will fix this shortly.",
      "PostedDate": "2010-09-10T14:05:44.65-07:00",
      "Id": -2147483648
    },
    {
      "Message": "",
      "PostedDate": "2010-11-15T12:21:44.887-08:00",
      "Id": -2147483648
    },
    {
      "Message": "rnwood,\r\n\r\nThis ticket is marked as fixed, do you know where I can get the updated version?\r\n\r\nthanks,\r\nSonny",
      "PostedDate": "2011-01-25T08:12:09.76-08:00",
      "Id": -2147483648
    },
    {
      "Message": "",
      "PostedDate": "2011-05-09T05:37:46.207-07:00",
      "Id": -2147483648
    },
    {
      "Message": "What's the deal? I got latest stable today and it still has this issue. I would love to be able to debug my python code with this app but can't right now.",
      "PostedDate": "2011-08-01T18:15:19.7-07:00",
      "Id": -2147483648
    },
    {
      "Message": "\r\n\r\n\n\n** Closed by rnwood 09/05/2011 05:37",
      "PostedDate": "2011-08-02T13:40:53.883-07:00",
      "Id": -2147483648
    },
    {
      "Message": "",
      "PostedDate": "2011-08-02T13:40:54.99-07:00",
      "Id": -2147483648
    },
    {
      "Message": "See 2.0.8.",
      "PostedDate": "2011-08-02T13:55:55.273-07:00",
      "Id": -2147483648
    },
    {
      "Message": "rnwood,\r\n\r\nThe same goes for commands, not only the parameters. Check this log, please:\r\n\r\nauth cram-md5\r\n504 Specified AUTH mechanism not supported\r\n\r\nIt is expecting CRAM-MD5, not cram-md5!\r\n\r\nthanks,\r\nVicente",
      "PostedDate": "2011-10-03T04:51:28.793-07:00",
      "Id": -2147483648
    },
    {
      "Message": "",
      "PostedDate": "2013-02-21T17:30:11.057-08:00",
      "Id": -2147483648
    },
    {
      "Message": "",
      "PostedDate": "2013-05-16T04:42:23.81-07:00",
      "Id": -2147483648
    },
    {
      "Message": "",
      "PostedDate": "2013-05-16T04:42:23.933-07:00",
      "Id": -2147483648
    },
    {
      "Message": "",
      "PostedDate": "2013-06-14T00:31:49.137-07:00",
      "Id": -2147483648
    }
  ]
}